fixtures/eurolite/led-tmh-x12.json

```
@@ -0,0 +1,564 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED TMH-X12",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["Flo Edelmann"],
+    "createDate": "2018-11-08",
+    "lastModifyDate": "2018-11-08"
+  },
+  "links": {
+    "manual": [
+      "https://media.steinigke.de/download_t/992-MANUAL-1.10-de-en_00110800.pdf"
```

```suggestion
      "https://media.steinigke.de/documents_t/992-MANUAL-1.10-de-en_00110800.pdf"
```

This will open the pdf inside the browser